.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: PyTables Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  pytables-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+      - name: 'Check out repo content'
+        uses: actions/checkout@v2
+      - name: 'Set up Python ${{ matrix.python-version }}'
+        uses: actions/setup-python@v2
+        with:
+          python-version: '${{ matrix.python-version }}'
+      # - run: sudo apt-get install libbz2-dev libhdf5-serial-dev liblzo2-dev
```

Comment:
Without this PyTables is not going to build

Reply:
Thanks, I added that line to the long list of `run` in `Run test`.